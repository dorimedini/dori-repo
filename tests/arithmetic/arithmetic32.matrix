{
	matrix m(2,3) = [1, 2, 3; 4, 5, 6];

	int a = 3;

	m=a*m;
	print m;
	m[0,0]=m[0,0]/3;
	m[0,1]=m[0,1]/3;
	m[0,2]=m[0,2]/3;
	m[1,0]=m[1,0]/3;
	m[1,1]=m[1,1]/3;
	m[1,2]=m[1,2]/3;
	print m;
	
}