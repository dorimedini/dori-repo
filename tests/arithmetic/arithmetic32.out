[3,6,9;12,15,18]
[1,2,3;4,5,6]
